neurogym/wrappers/monitor.py

```diff
@@ -149,3 +207,228 @@ def store_data(self, obs, action, rew, info) -> None:
             self.rew_mat = []
             self.gt_mat = []
             self.perf_mat = []
+
+    def plot_training_history(self, window_size=None, figsize=(12, 6), save_fig=True):
+        """Plot training history from saved data files with one data point per trial.
+
+        Args:
+            window_size: Size of the moving average window (default: auto-calculate based on data size)
```

Comment:
Maybe make explicit that the size is in number of trials, and what the auto-calculated default is.

```suggestion
            window_size: Number of trials for the moving average window (default: ~5% of total trials)
```

Reply:
I will remove the window completely